File: fizzbuzz.js
var fizzBuzz = function (num) {
  if (num % 15 === 0) {
    return "Fizzbuzz";
  } else if (num % 3 === 0) {
    return "Fizz";
  } else if (num % 5 === 0) {
    return "Buzz";
  } else {
    return num;
  }
};

console.log(fizzBuzz(8));

File: fizzbuzz.rb
def fizzbuzz(num)
  if num % 15 == 0
    puts "Fizzbuzz"
  elsif num % 5 == 0
    puts "Buzz"
  elsif num % 3 == 0
    puts "Fizz"
  else
    num
  end
end

puts fizzbuzz(1)
